spec/models/manageiq/providers/vmware/inventory/parser/network_manager_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe ManageIQ::Providers::Vmware::Inventory::Parser::NetworkManager do
+  describe "Utility" do
+    let(:parser) { described_class.new }
```

Comment:
There is a special variable for this in rspec, `subject`. Please remove this let and just use `subject` below instead `parser`.